languages/pt-br.ini

```
@@ -440,6 +440,7 @@ Exception = "Excepção"
 Excerpt = "Excerto"
 exclude_facet = "[excluir]"
 exclude_newspapers = "Excluir os artigos de jornal"
+excluded = "Excluídos:"
```

```suggestion
excluded = "Excluído:"
```
Updating translation per suggestion from @becahp